bpf/lib/nodeport.h

```diff
@@ -2709,6 +2716,10 @@ int tail_nodeport_nat_egress_ipv4(struct __ctx_buff *ctx)
 
 		src_port = tunnel_gen_src_port_v4(&tuple);
 
+#if !(defined(ENABLE_CLUSTER_AWARE_ADDRESSING) && defined(ENABLE_INTER_CLUSTER_SNAT))
```

Comment:
offline it was discussed to have the callers of `nodeport_lb4()` set the `src_sec_identity`, and then avoid messing with it inside the tail calls chain (@brb) but this needs to remain in `tail_nodeport_nat_egress_ipv4()` as overwriting the source identity with `WORLD_IPV4_ID` in the callers would break policy enforcement